cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -234,24 +234,19 @@ type annotator struct {
 	annotate func(*core.Node, *compute.Instance) bool
 }
 
+var (
+	nodeURLRegexpStr = "^gce://([^/]+)/([^/]+)/([^/]+)$"
+	nodeURLRegexp    = regexp.MustCompile(nodeURLRegexpStr)
```

Comment:
combine these

Reply:
I use the string in the error message. Want me to remove it?